docs/api-reference/webgl/copy-and-blit.md

```diff
@@ -0,0 +1,163 @@
+# Copy and Blit
+
+`luma.gl` offers a set of methods that copy or blit data from and to Texture and Framebuffer objects. Data can also be copied into Buffer, TypedArray, Images or Urls.
```

Comment:
* set of methods => set of functions (in my mind a method is a class function)
* data can also => Image data can also be copied from Framebuffers and Textures into...